src/Components/CollectionsHubRails/ArtistSeriesRail/ArtistSeriesEntity.tsx

```diff
@@ -0,0 +1,140 @@
+import { Box, color, Flex, Link, Sans, Serif } from "@artsy/palette"
+import { ArtistSeriesEntity_member } from "__generated__/ArtistSeriesEntity_member.graphql"
+import currency from "currency.js"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import styled from "styled-components"
+import { get } from "Utils/get"
+
+export interface ArtistSeriesEntityProps {
+  member: ArtistSeriesEntity_member
+}
+
+export const ArtistSeriesEntity: React.FC<ArtistSeriesEntityProps> = ({
+  member,
+}) => {
+  const {
+    headerImage,
+    artworks: { hits },
+    price_guidance,
+    slug,
+    title,
+  } = member
+  const bgImages = hits.map(hit => hit.image.url)
+  const imageSize =
+    bgImages.length === 1 ? 265 : bgImages.length === 2 ? 131 : 85
+  return (
+    <Container p={0.5} m={0.5}>
+      <StyledLink href={`${sd.APP_URL}/collection/${slug}`}>
+        <ImgWrapper pb={1}>
+          {bgImages.length ? (
+            bgImages.map((url, i) => {
+              const artistName = get(hits[i].artist, a => a.name)
+              const alt = `${artistName ? artistName + ", " : ""}${
+                hits[i].title
```

Comment:
I wonder if this should simply be the artwork title by the artist name if the artist name is present?

Something like: `${hits[i].title} by {artistName}`